src/collapse/collapse.js

```diff
@@ -29,24 +41,44 @@ angular.module('ui.bootstrap.collapse', [])
                 .attr('aria-expanded', true)
                 .attr('aria-hidden', false);
 
-              if ($animateCss) {
-                $animateCss(element, {
-                  addClass: 'in',
-                  easing: 'ease',
-                  to: { height: element[0].scrollHeight + 'px' }
-                }).start()['finally'](expandDone);
+              if (horizontal) {
```

Comment:
It probably would be worthwhile converting everything in here into a helper function to avoid duplication between horizontal and vertical collapsing.